templates/account/login.html

```diff
@@ -24,7 +24,7 @@
         {% include "socialaccount/snippets/login_extra.html" %}
 
         <div class="d-flex justify-content-center mb-4">
-          <div class="content-2">or</div>
+          <div class="content-2">{% trans_html "py.templates.account.login.socialSimpleSeparator" default="or" comment="As in 'Login with Google account or login with simple account'" %}</div>
```

Comment:
I didn't understand this comment when I saw it in the .pofiles. Can we try iterating? My brainstorm idea:

`"The 'or' in 'Log in with [Google/Facebook buttons] *or* [username/password]'"`